Directory.Build.props

```diff
@@ -15,6 +15,9 @@
     
     <!-- .NET 6+ packages support back to API-21 -->
     <SupportedOSPlatformVersion>21</SupportedOSPlatformVersion>
+    
+    <!-- Mark .NET6+ packages as supporting trimming -->
+    <IsTrimmable>true</IsTrimmable>
```

Comment:
Should we also remove this line?

https://github.com/xamarin/AndroidX/blob/c18c9766e921e96b69d65094b402648214d8a853/source/AssemblyInfo.cs#L13